### PR TITLE
IBX-7036: Customizable dashboard described in Developer documentation

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://issues.ibexa.co/browse/IBX-7036)
| Versions      | master
| Edition       | Content/Headless, Experience, Commerce

Customizable dashboard described in Developer documentation.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/administration/dashboard/php_api_dashboard_service.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: PHP API Dashboard service
+description: PHP API Dashboard service
+---
+
+# PHP API Dashboard service
```
Comment (adamwojs) [inline docs/administration/dashboard/php_api_dashboard_service.md:6]: We should either expand this section (it's only one sentence at the moment) or remove it.

**Review by adamwojs — APPROVED** [review]

File: docs/administration/dashboard/php_api_dashboard_service.md (hunk 1 above)
Comment (julitafalcondusza) [reply docs/administration/dashboard/php_api_dashboard_service.md:6]: It is draft PR, still in progress, section will be expanded.

File: docs/administration/dashboard/configure_default_dashboard.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+## Content Type identifier
+
+It is an identifier that represents dashboard Content Type.
+You can find it in the Admin panel, **Dashboard Content Type** section, **View/Global properties** tab.
+
+## Container Content Type identifier
+
+Specifies the Content Type of the container for configured dashboard, by default all the dashboards containers are set as a folders.
+
+![Container Content Type](dashboard_container_type.png)
```
Comment (adriendupuis) [inline docs/administration/dashboard/configure_default_dashboard.md:70]: To be confirmed bu it seems that this the content type for the container created under "User dashboards" each type a user create own custom dashboard.

File: docs/administration/dashboard/configure_default_dashboard.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+## Users container remote ID
+
+Defines a container for users folders, which contain all customized dashboards.
+You can see it in the Admin panel, **Dashboards** section, **User dashboards** folder inside of main **Dashboards** container in the Content Tree. 
+In the **Technical details** tab it is defined as **Location remote ID**.
+
+## Predefined container remote ID
+
+Defines a container that contains all predefined dashboards created by Administrator.
+You can see it in the Admin panel, **Dashboards** section, **User dashboards** folder inside of main **Dashboards** container in the Content Tree.
```
Comment (adriendupuis) [inline docs/administration/dashboard/configure_default_dashboard.md:54]:
```suggestion
You can see it in the Admin panel, **Dashboards** section, **Predefined dashboards** folder inside of main **Dashboards** container in the Content Tree. 
```

File: docs/administration/dashboard/configure_default_dashboard.md (hunk 2 above)
Comment (mikadamczyk) [reply docs/administration/dashboard/configure_default_dashboard.md:70]: It determines the content type identifier of the container which will be possible to create. e.g. if an administrator wants to create an additional structure to the predefined dashboards

image

File: docs/administration/dashboard/php_api_dashboard_service.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+You can use `DashboardService`'s PHP API to manage custom dashboards.
+To obtain this service, inject the `Ibexa\Contracts\Dashboard\DashboardServiceInterface`.
+
+The service exposes two functions:
+
+- `createCustomDashboardDraft(?Location $location = null): Content` - returns a new Content item in draft state of `dashboard` Content Type.
+  If no location is given, it creates by copying the current user's active dashboard.
+  If a location is given, it creates by copying the given location.
+  This new Content draft is located in the current user custom dashboard container.
```
Comment (mikadamczyk) [inline docs/administration/dashboard/php_api_dashboard_service.md:16]: The default name of the customized dashboard is set to 'My dashboard'.

**Review by mikadamczyk — APPROVED** [review]

File: docs/administration/dashboard/configure_default_dashboard.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+Configuration can be set per [SiteAccess](multisite/multisite_configuration/#siteaccess-configuration)
+or [SiteAccess group](multisite/multisite_configuration/#siteaccess-groups).
+
+All the settings in the configuration are reflected in the Back Office.
+
+## Container remote ID
+
+Defines starting location container for all the dashboards, including customized and predifined ones.
+You can see it in the Admin panel, **Dashboards** section, **Dashboards** folder in the Content Tree. In the **Technical details** tab it is defined as **Location remote ID**.
```
Comment (juskora) [inline docs/administration/dashboard/configure_default_dashboard.md:35]:
```suggestion
You can see it in the Admin panel, **Dashboards** section, **Dashboards** folder in the Content Tree. In the **Technical details** tab, it is defined as **Location remote ID**.
```

File: docs/administration/dashboard/configure_default_dashboard.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+![Container remote ID](dashboard_container_remote_id.png)
+
+## Default dashboard remote ID
+
+Specifies default predefined dashboard. This is the dashboard that all the users see as a starting dashboard in the Back Office.
+You can see it in the Admin panel, **Dashboards** section, **Default dashboard** folder inside of **Predefined dashboards** container in the Content Tree. 
+In the **Technical details** tab it is defined as **Location remote ID**.
```
Comment (juskora) [inline docs/administration/dashboard/configure_default_dashboard.md:43]:
```suggestion
In the **Technical details** tab, it is defined as **Location remote ID**.
```

File: docs/administration/dashboard/configure_default_dashboard.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+Defines a container for users folders, which contain all customized dashboards.
+You can see it in the Admin panel, **Dashboards** section, **User dashboards** folder inside of main **Dashboards** container in the Content Tree. 
+In the **Technical details** tab it is defined as **Location remote ID**.
+
+## Predefined container remote ID
+
+Defines a container that contains all predefined dashboards created by Administrator.
+You can see it in the Admin panel, **Dashboards** section, **Predefined dashboards** folder inside of main **Dashboards** container in the Content Tree. 
+In the **Technical details** tab it is defined as **Location remote ID**.
```
Comment (juskora) [inline docs/administration/dashboard/configure_default_dashboard.md:55]:
```suggestion
In the **Technical details** tab, it is defined as **Location remote ID**.
```

File: docs/administration/dashboard/configure_default_dashboard.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+Specifies the name of the [Section](administration/content_organization/sections/).
+
+## Content Type identifier
+
+It is an identifier that represents dashboard Content Type.
+You can find it in the Admin panel, **Dashboard Content Type** section, **View/Global properties** tab.
+
+## Container Content Type identifier
+
+Determines the Content Type identifier of the container for dashboards and let you create additional structure for the predefined dashboards.
```
Comment (juskora) [inline docs/administration/dashboard/configure_default_dashboard.md:68]:
```suggestion
Determines the Content Type identifier of the container for dashboards and lets you create additional structure for the predefined dashboards.
```

File: docs/administration/dashboard/configure_default_dashboard.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+In the **Technical details** tab it is defined as **Location remote ID**.
+
+## Section identifier
+
+Specifies the name of the [Section](administration/content_organization/sections/).
+
+## Content Type identifier
+
+It is an identifier that represents dashboard Content Type.
+You can find it in the Admin panel, **Dashboard Content Type** section, **View/Global properties** tab.
```
Comment (juskora) [inline docs/administration/dashboard/configure_default_dashboard.md:64]: Add a screenshot illustrating this identifier.

File: docs/administration/dashboard/customize_dashboard.md [hunk 10]
```diff
@@ -0,0 +1,39 @@
+---
+title: Dashboard customization
+description: Customize dashboard.
+---
+
+# Customize dashboard
+
+You can customize the dashboard depending on your needs using Dahboard Builder.
+Customized dashboard previews a set of widgets selected by the user.
```
Comment (juskora) [inline docs/administration/dashboard/customize_dashboard.md:9]:
```suggestion
Customized dashboard displays a set of widgets selected by the user.
```

File: docs/administration/dashboard/customize_dashboard.md [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+
+!!! tip
+
+     Detailed instruction on how to customize dashboard using Dashboard Builder is available in [[[= product_name =]] user documentation](https://doc.ibexa.co/projects/userguide/en/latest/getting_started/discover_ui/).
+
+## Manage permissions
+
+In order to be able to customize dashboard, you need to have `dashboard/cutomize` [policy](permission_overview.md).
```
Comment (juskora) [inline docs/administration/dashboard/customize_dashboard.md:17]:
```suggestion
To customize dashboard, you need to have `dashboard/cutomize` [policy](permission_overview.md).
```

File: docs/administration/dashboard/php_api_dashboard_service.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+# DashboardService's PHP API
+
+You can use `DashboardService`'s PHP API to manage custom dashboards.
+To obtain this service, inject the `Ibexa\Contracts\Dashboard\DashboardServiceInterface`.
+
+The service exposes two functions:
+
+- `createCustomDashboardDraft(?Location $location = null): Content` - returns a new Content item in draft state of `dashboard` Content Type.
+  If no location is given, it creates by copying the current user's active dashboard.
+  If a location is given, it creates by copying the given location.
```
Comment (juskora) [inline docs/administration/dashboard/php_api_dashboard_service.md:15]: Rewrite.

File: docs/administration/dashboard/php_api_dashboard_service.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+  The default name of the customized dashboard is set as `My dashboard`.
+  This new Content draft is located in the current user custom dashboard container.
+- `createDashboard(DashboardCreateStruct $dashboardCreateStruct): Content` - publishes the given dashboard creation structure (`Ibexa\Contracts\Dashboard\Values\DashboardCreateStruct`) under `dashboard.predefined_container_remote_id`.
+
+## Customize dashboard using DashboardService
+
+The following example is a command deploying a custom dashboard to users of content groups.
+It loads the users of the groups using `admin` account, then it logs each user in, sets a custom dashboard by copying a model into a draft,
+and publishes this dashboard draft.
```
Comment (juskora) [inline docs/administration/dashboard/php_api_dashboard_service.md:24]: Rewrite.

**Review by juskora — CHANGES_REQUESTED** [review]

File: docs/administration/dashboard/configure_default_dashboard.md [hunk 14]
```diff
@@ -0,0 +1,73 @@
...
+
+Defines a container that contains all predefined dashboards created by Administrator.
+You can see it in the Admin panel, **Dashboards** section, **Predefined dashboards** folder inside of main **Dashboards** container in the Content Tree.
+In the **Technical details** tab, it is defined as **Location remote ID**.
+
+## Section identifier
+
+Specifies the name of the [Section](administration/content_organization/sections/).
+
+## Content type identifier
```
Comment (adriendupuis) [inline docs/administration/dashboard/configure_default_dashboard.md:61]:
```suggestion
## Content Type identifier
```

File: docs/administration/dashboard/configure_default_dashboard.md [hunk 15]
```diff
@@ -0,0 +1,73 @@
...
+
+It is an identifier that represents dashboard content type.
+You can find it in the Admin panel, **Dashboard content Ttpe** section, **View/Global properties** tab.
+
+![Content type identifier](dashboard_content_type_identifier.png)
+
+## Container content type identifier
```
Comment (adriendupuis) [inline docs/administration/dashboard/configure_default_dashboard.md:68]:
```suggestion
## Container Content Type identifier
```

File: docs/administration/dashboard/configure_default_dashboard.md [hunk 16]
```diff
@@ -0,0 +1,73 @@
...
+
+Determines the content type identifier of the container for dashboards and lets you create additional structure for the predefined dashboards.
```
Comment (adriendupuis) [inline docs/administration/dashboard/configure_default_dashboard.md:70]:
```suggestion
Determines the Content Type identifier of the container for dashboards and lets you create additional structure for the predefined dashboards.
```

File: docs/administration/dashboard/configure_default_dashboard.md [hunk 17]
```diff
@@ -0,0 +1,73 @@
...
+By default all the dashboards containers are set as a folders.
+
+![Container content type](dashboard_container_type.png)
```
Comment (adriendupuis) [inline docs/administration/dashboard/configure_default_dashboard.md:73]:
```suggestion
![Container Content Type](dashboard_container_type.png)
```

File: docs/administration/dashboard/configure_default_dashboard.md (hunk 14 above)
Comment (julitafalcondusza) [reply docs/administration/dashboard/configure_default_dashboard.md:61]: Capitalization removed as discussed with @juskora.

File: docs/administration/dashboard/configure_default_dashboard.md (hunk 15 above)
Comment (julitafalcondusza) [reply docs/administration/dashboard/configure_default_dashboard.md:68]: Capitalization removed as discussed with @juskora.

File: docs/administration/dashboard/configure_default_dashboard.md (hunk 16 above)
Comment (julitafalcondusza) [reply docs/administration/dashboard/configure_default_dashboard.md:70]: As above.

File: docs/administration/dashboard/configure_default_dashboard.md (hunk 17 above)
Comment (julitafalcondusza) [reply docs/administration/dashboard/configure_default_dashboard.md:73]: As above.

**Review by adriendupuis — APPROVED** [review]

**Pull request merged**